identifiers/country-hk/functional_constituencies.csv

```diff
@@ -9,7 +9,7 @@ ocd-division/country:hk/functional_constituency:accountancy,Accountancy
 ocd-division/country:hk/functional_constituency:medical,Medical
 ocd-division/country:hk/functional_constituency:health_services,Health Services
 ocd-division/country:hk/functional_constituency:engineering,Engineering
-ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,Architectural; Surveying and Planning (later on Architectural; Surveying; Planning and Landscape)
+ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,"Architectural, Surveying and Planning (later on Architectural, Surveying, Planning and Landscape)"

```

Comment:
Why the "Name (later Name)" format? Why not just the current name (as in my previous suggestion)?

Reply:
Same as above.